web-app/helpers/certificate-designs.ts

```diff
@@ -27,20 +27,22 @@ function addText(canvas: Canvas, text: string, font: string, fillStyle: string,
   context.fillText(text, leftPos, rightPos);
```

Comment:
I was probably the person who wrote this line. Now that I'm looking at it again, `rightPos` felt like a total mistake of a name.

So I double-checked https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/fillText and saw what I expected, which is that that argument refers to the vertical positioning.

In this addText function, can you please rename the arguments from leftPos to xPos and from rightPos to yPos?